e2e/features/data-download/data-download-test.js

```diff
@@ -0,0 +1,99 @@
+const reuseables = require('../../reuseables/skip-tour.js');
+const layersTab = '#layers-sidebar-tab';
+const dataDownloadTabButton = '#download-sidebar-tab';
+const zoomInButton = '#wv-map-geographic > div.wv-map-zoom-in.wv-map-zoom.ui-button.ui-corner-all.ui-widget';
+const zoomOutButton = '#wv-map-geographic > div.wv-map-zoom-out.wv-map-zoom.ui-button.ui-corner-all.ui-widget';
+
+const TIME_LIMIT = 20000;
+module.exports = {
+  'Initial State - Data Download tab is available and in default state when clicked': function(client) {
+    reuseables.loadAndSkipTour(client, TIME_LIMIT);
+
+    // Verify Data Download tab is in sidebar
+    client.expect.element(dataDownloadTabButton).to.be.visible;
+
+    // Click Data Download tab to switch to Data with 'No Data Selected'
+    client.click(dataDownloadTabButton);
+    client.expect.element('#compare-toggle-button > span').to.have.text.equal('No Data Selected');
+
+    // All 6 default layers in Not Available for Download
+    client.elements('css selector', '.wv-datacategory > li', function(result) {
+      client.assert.equal(result.value.length, 6);
+    });
+
+    // No 'Searching for Data' CMR query indicator present
+    client.expect.element('#indicator').to.not.be.present;
+  },
+  'No Results - No Data Available indicator displayed when no data': function(client) {
+    // December 31, 2022 no results
+    let queryString = '?now=2022-12-31T12&p=geographic&l=MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor,MODIS_Terra_Aerosol,Coastlines&t=2022-12-31&v=-91.125,-53.3671875,82.40625,59.8359375&download=MOD04_L2';
+    client.url(client.globals.url + queryString);
+
+    // 'No Data Available' indicator present
+    client.expect.element('#indicator').to.be.present.after(TIME_LIMIT);
+    client.expect.element('#indicator > span').to.have.text.equal('No Data Available').after(500);
+
+    // On Valid Data Select, 'No Data Available' indicator disappears and data layer is in sidebar
+    client.clearValue('#year-animation-widget-main');
+    client.setValue('#year-animation-widget-main', ['2013', client.Keys.ENTER]);
+    client.expect.element('#indicator').to.not.be.visible;
+    client.expect.element('#wv-data-MOD04_L2MODIS_Terra_Aerosol').to.be.visible;
+
+    // On Valid Data Select, 'No Data Available' indicator appears
+    client.clearValue('#year-animation-widget-main');
+    client.setValue('#year-animation-widget-main', ['2022', client.Keys.ENTER]);
```

Comment:
Looks like this `setValue()` is breaking `IE` and `Safari`.

Reply:
It appears that the `inputValue` array with `client.Keys.ENTER` is causing the issue on some browsers. I also tried changing the input values and pressing `TAB` after setting the value, but that didn't work either. I refactored with a naive solution that clicks the up/down arrows on the timeline to change dates. Let me know if you think this will work for us.

Reply:
Seems like a good solution  👍